frontend/src/scenes/sessions/sessionsTableLogic.ts

```diff
@@ -176,8 +186,17 @@ export const sessionsTableLogic = kea<sessionsTableLogicType<SessionRecordingId>
                 ),
         ],
         expandedRowKeys: [
-            (selectors) => [selectors.sessions],
-            (sessions: SessionType[]): string[] => sessions?.map((s) => s.global_session_id) || [],
+            (selectors) => [selectors.sessions, selectors.rowExpandState],
+            (sessions: SessionType[], rowExpandState: ExpandState): string[] | boolean => {
```

Comment:
`string[] | boolean` is such a weird type and does not match with the name -> code smell.

Would empty array work for the 2nd case?

Alternatively how about renaming this `expandedRowKeysProps` and returning either `{}` or `{ expandedRowKeys: [1,2,3] }` - this avoids weird hacks in the component.

Reply:
Thanks! that makes sense. Still kind of hacky that I'm inferring state from the "emptiness" of an object. Maybe I can also pass in a `state: ExpandState` key into this object to be more explicit?

Reply:
Ah I see what you're getting at! super clean that we can just drop this into `expandable`